internal/pkg/service/common/etcdop/watch.go

```diff
@@ -6,31 +6,70 @@ import (
 	"go.etcd.io/etcd/api/v3/etcdserverpb"
 	"go.etcd.io/etcd/api/v3/mvccpb"
 	etcd "go.etcd.io/etcd/client/v3"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type EventType int32
+
+const (
+	CreateEvent EventType = iota
+	UpdateEvent
+	DeleteEvent
```

Comment:
5. Pridal som abstrakciu `GetAllAndWatch`,
kedze to je myslim ten use-case, ktory riesime.

Nerobim to aby som to komplikoval ale naopak `Abstraction aims to hide as much complexity as possible to allow programmers to focus on what's most important`.

Takto sa to da pekne otestovat a jednoduchsie sa bude hladat chyba v kode, ked vieme, ze toto mame spravne.